flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -432,24 +432,42 @@ def chart_for_multiple_sensors(
 
     sensors_specs = []
     for s in sensors_to_show:
+        custom_title = None
         # List the sensors that go into one row
         if isinstance(s, list):
             row_sensors: list["Sensor"] = s  # noqa F821
+        elif isinstance(s, dict):
+            # Set up custom title for sensor
+            custom_title = s["title"]
+            if "sensors" in s:
+                row_sensors: list["Sensor"] = s["sensors"]  # noqa F821
+            else:
+                row_sensors: list["Sensor"] = [s["sensor"]]  # noqa F821
         else:
             row_sensors: list["Sensor"] = [s]  # noqa F821
 
         # Derive the unit that should be shown
         unit = determine_shared_unit(row_sensors)
         sensor_type = determine_shared_sensor_type(row_sensors)
 
-        # Set up field definition for event values
-        event_value_field_definition = dict(
-            title=f"{capitalize(sensor_type)} ({unit})",
-            format=[".3~r", unit],
-            formatType="quantityWithUnitFormat",
-            stack=None,
-            **FIELD_DEFINITIONS["event_value"],
-        )
+        if custom_title:
+            # Set up field definition for event values
+            event_value_field_definition = dict(
+                title=f"{capitalize(custom_title)} ({unit})",
```

Comment:
Like this ? ![Screenshot from 2024-08-29 15-31-15](https://github.com/user-attachments/assets/be5829e0-8098-4757-9d3c-a72a7a627456)

Reply:
Exactly!

Reply:
Do you have any suggestions for graphs we don't specify title for , since we want to keep support for sensors input without titles : 

[  1, [2,3], 
  { "title": "row1 Title","sensor": 4},
  { "title": "row2 Title", "sensors": [5, 6]}
]
Should we leave in the graph an empty title or put a place holder title?

Reply:
I don't have a good idea for a placeholder, so I vote for empty.